src/components/EnergyConsumptionChart.tsx

```diff
@@ -1,241 +1,264 @@
-        fill="text"
-        width={50}
-        textAnchor="middle"
-        verticalAnchor="middle"
-        fontSize="2xs"
-      >
-        {payload.value}
-      </RechartText>
-    </g>
-  )
-}
+// ChartDataLabels required to display y-labels on top of bars
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  Legend,
+  BarElement,
+  ChartDataLabels
+)
 
 const EnergyConsumptionChart = () => {
   const { t } = useTranslation("page-what-is-ethereum")
-  const textColor = useToken("colors", "text")
   const { locale } = useRouter()
   const isRtl = isLangRightToLeft(locale as Lang)
 
-  const data = useBreakpointValue<Data>({
-    base: [
-      {
-        name: t("energy-consumption-chart-global-data-centers-label"),
-        amount: 190,
-        color: "#FF0000",
-      },
-      {
-        name: t("energy-consumption-chart-btc-pow-label"),
-        amount: 149,
-        color: "#F2A900",
-      },
-      {
-        name: t("energy-consumption-chart-gaming-us-label"),
-        amount: 34,
-        color: "#71BB8A",
-      },
-      {
-        name: t("energy-consumption-chart-eth-pow-label"),
-        amount: 21,
-        color: "#C1B6F5",
-      },
-      {
-        name: t("energy-consumption-chart-eth-pos-label"),
-        amount: 0.0026,
-        color: "#C1B6F5",
-      },
-    ],
-    sm: [
-      {
-        name: t("energy-consumption-chart-global-data-centers-label"),
-        amount: 190,
-        color: "#FF0000",
-      },
-      {
-        name: t("energy-consumption-chart-btc-pow-label"),
-        amount: 149,
-        color: "#D7B14A",
-      },
-      {
-        name: t("energy-consumption-gold-mining-cbeci-label"),
-        amount: 131,
-        color: "#F2A900",
-      },
-      {
-        name: t("energy-consumption-chart-eth-pow-label"),
-        amount: 21,
-        color: "#C1B6F5",
-      },
-      {
-        name: t("energy-consumption-chart-netflix-label"),
-        amount: 0.457,
-        color: "#E50914",
-      },
-      {
-        name: t("energy-consumption-chart-eth-pos-label"),
-        amount: 0.0026,
-        color: "#C1B6F5",
-      },
-    ],
-    md: [
-      {
-        name: t("energy-consumption-chart-global-data-centers-label"),
-        amount: 190,
-        color: "#FF0000",
-      },
-      {
-        name: t("energy-consumption-chart-btc-pow-label"),
-        amount: 149,
-        color: "#D7B14A",
+  // chart labels
+  const labels = [
+    t("energy-consumption-chart-global-data-centers-label"),
+    t("energy-consumption-chart-btc-pow-label"),
+    t("energy-consumption-gold-mining-cbeci-label"),
+    t("energy-consumption-chart-gaming-us-label"),
+    t("energy-consumption-chart-eth-pow-label"),
+    "Google",
+    t("energy-consumption-chart-netflix-label"),
+    t("energy-consumption-chart-paypal-label"),
+    t("energy-consumption-chart-airbnb-label"),
+    t("energy-consumption-chart-eth-pos-label"),
+  ]
+
+  const chartOptions = {
+    // chart styles
+    barThickness: 38,
```

Comment:
Wondering if we could pass a responsive Chakra token here to make the width responsive... or perhaps we could use a maxBarThickness here to cap it on Mobile.